### ISSUE TITLE
Unity Ads Adapter is not working as expected

### ISSUE THREAD

**Issue by Izzyjm** [googleads/googleads-mobile-ios-mediation#125]
The first banner request shows fine, but then once it makes another request after 30 seconds it shows this each time it makes a request.  I don't know why its calling "Unity Ads Banner did hide." it seems like the unity ad is loading but for some reason its then getting hidden and then destroyed and this happens each time the ad is requested it doesn't even get Ad Failed and then load an ad from another network in the mediation stack.  It keeps repeating this same thing while the banner is hidden

```
I/UnityAds: +[UnityAdsBanner loadBanner:] (line:19) :: Unity Ads opening banner ad unit for placement Banner
Unity Ads Banner did hide.
I/UnityAds: +[UnityAdsBanner destroy] (line:41) :: Unity Ads destroying current banner ad unit.
Unity Ads Banner is showing.
Unity Ads Banner did hide.
Unity Ads Banner has unloaded.
```

![Screen Shot 2019-06-18 at 12 49 00 AM](https://user-images.githubusercontent.com/24462583/59656474-4c55ec00-9163-11e9-82b6-f9a2349572f3.png)

**Comment by VNKorchagin** [comment]
I have the same problem

  - GoogleMobileAdsMediationUnity (3.1.0.0):
    - Google-Mobile-Ads-SDK (>= 7.42.2)
    - UnityAds (= 3.1.0)

**Comment by joshuajplagonera** [comment]
Hi all,
Thank you for your patience.
A fix has been released for this issue on version 3.2.0.0 of the Unity iOS adapter. Kindly let us know if you are still experiencing the issue with the new version.

**Comment by VNKorchagin** [comment]
Issue still reproducible

  - GoogleMobileAdsMediationUnity (3.2.0.1):
    - Google-Mobile-Ads-SDK (>= 7.49.0)
    - UnityAds (= 3.2.0)

Logs:

> 2019-09-17 16:42:46.781633+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner destroy] (line:41) :: Unity Ads destroying current banner ad unit.
2019-09-17 16:42:46.795104+0300 [APPNAME HIDDEN][1727:224684] D/UnityAds: -[USRVWebViewMethodInvokeOperation main] (line:32) :: Locking
2019-09-17 16:42:46.804755+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityServices initialize:delegate:testMode:usePerPlacementLoad:] (line:50) :: Initializing Unity Ads 3.2.0 (3200) with game id HIDDEN_GAME_ID in production mode
2019-09-17 16:42:46.844179+0300 [APPNAME HIDDEN][1727:224355] I/UnityAds: -[USRVInitializeStateConfig execute] (line:167) :: Unity Ads init: load configuration from https://config.unityads.unity3d.com/webview/3.2.0/release/config.json
2019-09-17 16:42:47.097171+0300 [APPNAME HIDDEN][1727:224355] I/UnityAds: -[USRVInitializeStateLoadCache execute] (line:205) :: Unity Ads init: webapp loaded from local cache
2019-09-17 16:42:48.037567+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[USRVApiSdk WebViewExposed_logInfo:callback:] (line:67) :: Requesting configuration from https://publisher-config.unityads.unity3d.com/games/[HIDDEN_GAME_ID]/configuration?bundleId=[HIDDEN_BUNDLE_ID]&encrypted=true&rooted=false&platform=ios&sdkVersion=3200&osVersion=12.0.1&deviceModel=iPhone7%2C2&language=ru_RU&test=false&advertisingTrackingId=[HIDDEN_INFO]&limitAdTracking=false
2019-09-17 16:42:48.360533+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[USRVApiSdk WebViewExposed_logInfo:callback:] (line:67) :: Received configuration for token [HIDDEN_TOKEN] (A/B group 0)
2019-09-17 16:42:48.428696+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner loadBanner:] (line:19) :: Unity Ads opening banner ad unit for placement [PLACEMENT_HIDDEN]
2019-09-17 16:42:48.493948+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[USRVApiSdk WebViewExposed_logInfo:callback:] (line:67) :: Requesting ad plan from https://auction.unityads.unity3d.com/v5/games/[HIDDEN_GAME_ID]/requests?advertisingTrackingId=[HIDDEN_TRACKING_ID]&limitAdTracking=false&deviceModel=iPhone7%2C2&platform=ios&sdkVersion=3200&stores=apple&osVersion=12.0.1&screenScale=2&screenWidth=375&screenHeight=667&connectionType=wifi&networkType=13
2019-09-17 16:42:50.881654+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[USRVApiSdk WebViewExposed_logInfo:callback:] (line:67) :: AdPlan received with 1 campaigns and refreshDelay 0
2019-09-17 16:42:51.799198+0300 [APPNAME HIDDEN][1727:224684] E/UnityAds: -[USRVWebViewMethodInvokeOperation main] (line:37) :: Unity Ads callback timed out! UADSWebViewBannerHideOperation callback:
2019-09-17 16:42:54.242691+0300 [APPNAME HIDDEN][1727:224150] Unity Ads Banner is showing.
2019-09-17 16:42:54.719326+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner destroy] (line:41) :: Unity Ads destroying current banner ad unit.
2019-09-17 16:42:54.719599+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner loadBanner:] (line:19) :: Unity Ads opening banner ad unit for placement [PLACEMENT_HIDDEN]
2019-09-17 16:42:54.754510+0300 [APPNAME HIDDEN][1727:224150] Unity Ads Banner did hide.
2019-09-17 16:42:54.754554+0300 [APPNAME HIDDEN][1727:224150] Unity Ads Banner has unloaded.
2019-09-17 16:42:55.162315+0300 [APPNAME HIDDEN][1727:224150] E/UnityAds: +[USRVApiSdk WebViewExposed_logError:callback:] (line:57) :: No fill for placement [PLACEMENT_HIDDEN]
2019-09-17 16:42:55.164425+0300 [APPNAME HIDDEN][1727:224150] W/UnityAds: +[USRVApiSdk WebViewExposed_logWarning:callback:] (line:62) :: Could not show banner due to No fill for placement [PLACEMENT_HIDDEN]
2019-09-17 16:42:59.067279+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner destroy] (line:41) :: Unity Ads destroying current banner ad unit.
2019-09-17 16:42:59.067535+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner loadBanner:] (line:19) :: Unity Ads opening banner ad unit for placement [PLACEMENT_HIDDEN]
2019-09-17 16:42:59.418132+0300 [APPNAME HIDDEN][1727:224150] E/UnityAds: +[USRVApiSdk WebViewExposed_logError:callback:] (line:57) :: No fill for placement [PLACEMENT_HIDDEN]
2019-09-17 16:42:59.419866+0300 [APPNAME HIDDEN][1727:224150] W/UnityAds: +[USRVApiSdk WebViewExposed_logWarning:callback:] (line:62) :: Could not show banner due to No fill for placement [PLACEMENT_HIDDEN]
2019-09-17 16:43:03.395296+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner destroy] (line:41) :: Unity Ads destroying current banner ad unit.
2019-09-17 16:43:03.395575+0300 [APPNAME HIDDEN][1727:224150] I/UnityAds: +[UnityAdsBanner loadBanner:] (line:19) :: Unity Ads opening banner ad unit for placement [PLACEMENT_HIDDEN]
2019-09-17 16:43:04.535514+0300 [APPNAME HIDDEN][1727:224150] E/UnityAds: +[USRVApiSdk WebViewExposed_logError:callback:] (line:57) :: No fill for placement [PLACEMENT_HIDDEN]
2019-09-17 16:43:04.555122+0300 [APPNAME HIDDEN][1727:224150] W/UnityAds: +[USRVApiSdk WebViewExposed_logWarning:callback:] (line:62) :: Could not show banner due to No fill for placement [PLACEMENT_HIDDEN]
